src/test/java/net/masterthought/cucumber/generators/AbstractPageTest.java

```diff
@@ -66,7 +66,7 @@ public void generateReport_DisplaysContentAsEscapedText() {
         assertThat(document.getFeature().getElements()[0].getStepsSection().getSteps()[5].getEmbedding()[3].text())
                 .isEqualTo("Attachment 4 (HTML) <i>Hello</i> <b>World!</b>");
         assertThat(document.getFeature().getElements()[0].getStepsSection().getSteps()[5].getMessage().text())
-                .isEqualTo("Error message java.lang.AssertionError: \n" +
+                .isEqualTo("java.lang.AssertionError: java.lang.AssertionError: \n" +
```

Comment:
Input data was chnged in 14th line in steps.vm and in getErrorMessageTitle() inside Result class.